src/styles/blocks/product.scss

```diff
@@ -0,0 +1,55 @@
+.product {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 17px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &-image {
+    width: 160px;
+    height: 134px;
+    margin-left: 3px;
```

Comment:
Remove 'margin-left: 3px;' as the image should be centered horizontally within the product card.